### DISCUSSION TITLE
Library tab is missing on Settings Page

### DISCUSSION THREAD

**Discussion by computrav** [photoprism/photoprism#4535] [Q&A] [answered]
### GOAL
I'm would like to disable the Quality filter (by disabling the Review feature), which per the [User Guide -> Features -> Review page](https://docs.photoprism.app/user-guide/organize/review/) can be done on the Settings page. This is also mentioned on various other help posts about this issue. I'm aware that the setting is supposed to appear on the Library tab of the Settings page.

### ISSUE
I do not have the Library tab on the Settings page (https://docs.photoprism.app/user-guide/settings/library/).

**Screenshot:**
![image](https://github.com/user-attachments/assets/b80581a9-5a01-42ae-afa9-1a914ed71b71)

### WHAT I'VE DONE ALREADY
- Enabled 'Experimental Features' from the Advanced tab of the Settings page
- Restarted PhotoPrism via the UI
- Completely restarted the Docker containers (I am using Docker Compose).

Is there something else that could cause the Library tab in Settings to not show up?

**On a related note (documentation bug)**: links to the [User Guide about the Settings page](https://docs.photoprism.app/user-guide/settings/general/) makes no mention of the Review functionality or the Quality setting, so when you're directed there, it's quite confusing. I believe it should instead be linking to the page for the [Library tab within the Settings page]

**Comment by lastzero** [comment, marked as answer]
Your user doesn't seem to be super admin.

**Reply by computrav** [reply]
Hi - I can confirm that seems to have been the issue. I was using a 2nd account I had setup which was only a regular admin.

**If I can make a suggestion to avoid this confusion for others, it would be to make some notes about this scenario/requirement on the following User Guide pages:**
1. [Settings -> Library](https://docs.photoprism.app/user-guide/settings/library/)
2. [Managing Users -> Account Roles](https://docs.photoprism.app/user-guide/settings/library/)

In particular, if it were on the Library page of the User Guide right there at the top (something like, "This page is only accessible by Super Admins.", with a link to the Account Roles page, my question would have been answered immediately.

Thank you!

**Reply by graciousgrey** [reply]
Thank you for your feedback! 

I have just added a note to the [settings docs](https://docs.photoprism.app/user-guide/settings/library/)
